### ISSUE TITLE
RuntimeError: maximum recursion depth exceeded in cmp

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1347]
```
Exception occurred:
  File "/usr/lib/python2.7/site-packages/jinja2/utils.py", line 396, in __getitem__
    if self._queue[-1] != key:
RuntimeError: maximum recursion depth exceeded in cmp
The full traceback has been saved in /usr/src/tmp/sphinx-err-NjicN7.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-users/>,
or report them in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1347
- Originally reported by: Anonymous
- Originally created at: 2014-01-15T05:30:37.544

**Comment by shimizukawa** [comment]
_From Eugeny Rostovtsev on 2014-01-15 04:52:18+00:00_

Full build log:
http://git.altlinux.org/tasks/111887/build/100/x86_64/log

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-15 05:56:54+00:00_

This looks like a recursion looking for parent templates.

[Takayuki Shimizukawa](https://bitbucket.org/shimizukawa), can this be caused by https://bitbucket.org/birkenfeld/sphinx/pull-request/181?

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-01-15 06:07:46+00:00_

[Georg Brandl](https://bitbucket.org/birkenfeld) Yes. I think this issue is same as #1335 and it was fixed after 1.2.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-15 06:16:49+00:00_

Ah, sorry, I didn't recall this fix.  So much the better.

[Eugeny Rostovtsev](https://bitbucket.org/real_altlinux) this will be fixed in 1.2.1, which is due shortly.
